theia-ide/theia
# always use passed args for plugins tree actions

fix #5486

#### How to test

- install https://github.com/akosyakov/theia-vscode-gaps from sources (clone under plugins folder)
- run theia
- check that inline and context menu actions for tree items get proper arguments
- compare to VS Code if not sure
